pyproject.toml

```
@@ -0,0 +1,89 @@
+[project]
+name = "xsar"
+requires-python = ">= 3.10"
+license = { text = "MIT" }
+description = "Python xarray library to use Level-1 GRD Synthetic Aperture Radar products"
+readme = "README.md"
+dependencies = [
+  'GDAL',
+  'dask[array]',
+  'distributed',
+  "xarray>=2024.10.0",
+  'affine',
+  'rasterio',
+  'cartopy',
+  'fiona',
+  'pyproj',
+  'numpy',
+  'scipy',
+  'shapely',
+  'geopandas',
+  'fsspec',
+  'aiohttp',
+  'pytz',
+  'psutil',
+  'jinja2',
+  'rioxarray',
+  'lxml',
+]
+keywords = [
+  "xarray",
+  "earth-observation",
+  "remote-sensing",
+  "satellite-imagery",
+  "Sentinel-1",
+  "RCM",
+  "RadarSat2",
+  "sar",
+  "synthetic-aperture-radar",
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Intended Audience :: Science/Research",
+  "Topic :: Scientific/Engineering",
+]
+dynamic = ["version"]
+
+[project.urls]
+homepage = "https://github.com/umr-lops/xsar.readthedocs.io"
+documentation = "https://xsar.readthedocs.io"
+repository = "https://github.com/umr-lops/xsar"
+changelog = "https://xsar.readthedocs.io/en/latest/changelog.html"
+
+[project.scripts]
+ceos-alos2-create-cache = "ceos_alos2.sar_image.cli:main"
+xarray.backends = "xsar.xarray_backends:XsarXarrayBackend"
+
```

For extras, you can to this:
```suggestion

[project.optional-dependencies]
RS2 = ["xradarsat2"]
RCM = ["xarray-safe-rcm"]
S1 = ["xarray-safe-s1"]
```